tests/wp-tests-bootstrap.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * WordPress PHPUnit bootstrap file.
+ */
+
+namespace HM\Tests\Phpunit;
+
+$_root_dir = getcwd();
```

Comment:
I think this should be pointed one directory higher. Right now this is making everything relative to the `/tests/` directory.

Reply:
Replaces this with `dirname( __DIR__ )` which resolves to the parent directory of tests (the plugin's directory).